app/styles/_library/components/_whats-on.scss

```diff
@@ -0,0 +1,350 @@
+      background-repeat: no-repeat;
+    }
+
+    .whats-on-episode-segments .whats-on-episode-segment .whats-on-episode-segment-text,
+    .whats-on-episode-description .whats-on-episode-description-text {
+      color: rgb(var(--color-text));;
+      font-family: Open Sans;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: normal;
+      line-height: 16px;
+
+      @include media(">=medium") {
+        font-size: 18px;
+        line-height: 25px;
+      }
+    }
+
+    .whats-on-episode-segments {
+      align-items: center;
+      background-position: 4px top;
+      background-size: 2px 100%;
+      display: flex;
+      flex-direction: column;
+      padding-bottom: 10px;
+      padding-top: 10px;
+
+      @include media(">=medium") {
+        background-position: 7px top;
+      }
+
+      .whats-on-episode-segment {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        padding-bottom: 10px;
+        padding-top: 10px;
+        width: 100%;
+
+        .whats-on-episode-segment-dot {
+          background-color: #EAEFF0;
+          border-radius: 5px;
+          height: 10px;
+          margin-right: 10px;
+          min-width: 10px;
+
+          @include media(">=medium") {
+            border-radius: 8px;
+            height: 16px;
+            margin-right: 10px;
+            min-width: 16px;
+          }
+        }
+      }
+    }
+
+    .whats-on-episode-description {
+      align-items: center;
+      background-position: 7px top;
+      background-size: 2px 50%;
+      display: flex;
+      flex-direction: row;
+      padding-bottom: 20px;
+      padding-top: 20px;
+
+      .whats-on-episode-description-dot {
+        background-color: rgb(var(--color-primary-1));
+        border-radius: 8px;
+        height: 16px;
+        margin-right: 10px;
+        min-width: 16px;
+      }
+    }
+  }
+
+  .whats-on-schedule {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+
+    .whats-on-schedule-metadata {
+      align-items: center;
+      background-color: rgb(var(--color-background-muted));
+      display: flex;
+      flex-direction: row;
+      padding: 10px;
+
+      .whats-on-schedule-metadata-image {
+        background: rgb(var(--color-background-muted));
+        filter: drop-shadow(0px 1px 1px rgba(0, 0, 0, 0.15));
+        margin-right: 10px;
+        max-width: 48px;
+        min-height: 48px;
+        min-width: 48px;
+
+        @include media(">=medium") {
+          min-height: 72px;
+          min-width: 72px;
+        }
+      }
+
+      .whats-on-schedule-metadata-text-outer {
+        flex: 1;
+        display: flex;
+        flex-direction: row;
+
+        .whats-on-schedule-metadata-text-outer-title-outer {
+          flex: 1;
+
+          .whats-on-schedule-metadata-text-outer-title-outer-label {
+            color: rgb(var(--color-primary-1));
+            font-family: Open Sans;
+            font-size: 11.237px;
```

Comment:
Even worse if we can't stick to our font scale tokens. I would definitely ask design about these.

Reply:
yep I've already got a question out to design for this